### PR TITLE
Feat/scrolling profile view

### PR DESCRIPTION
**Pull request by ravibazz**
## Changes
- added scrolling view for profile videos.

### CODE REVIEW

File: src/component/back_btn.rs [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
 
 #[component]
-pub fn BackButton(fallback: String) -> impl IntoView {
-    let history_ctx = expect_context::<HistoryCtx>();
-
+pub fn BackButton() -> impl IntoView {
     let go_back = Callback::new(move |_| {
-        let back_url = history_ctx.back(&fallback);
-        use_navigate()(&back_url, Default::default());
+        let history = BrowserHistory::new();
```
Comment (rupansh-gob) [inline src/component/back_btn.rs:8]: how do we handle the fallback case?
If the user navigates from google.com to, say /refer-earn
where will they be redirected?

**Review by rupansh-gob — CHANGES_REQUESTED** [review]
looks good, just improve the history handling

File: src/component/back_btn.rs (hunk 1 above)
Comment (ravibazz) [reply src/component/back_btn.rs:8]: There isn't a way without using context and if we use context it ignores navigateOptions passed there is no way to track that

Comment (ravibazz) [reply src/component/back_btn.rs:8]: Do you think we should raise an issue with leptos that`use_location` should also return Navigate options

Comment (ravibazz) [reply src/component/back_btn.rs:8]: It is not possible to even  use that. Callback does not give any params to determine if the navigated url replaced the existing stack.

**Review by rupansh-gob — APPROVED** [review]

**Pull request merged**